dipy/reconst/dti.py

```diff
@@ -673,6 +674,53 @@ def tensor_prediction(dti_params, gtab, S0):
     return np.exp(np.dot(lower_tri, D.T))
 
 
+def adjacency_calc(img_shape, mask=None, distance=1.99):
+    """Create adjacency list for voxels, accounting for the mask.
+
+    Parameters
+    ----------
+
+    img_shape : list
+        Spatial shape of the image data.
+
+    mask : array, optional
+        A boolean array used to mark the coordinates in the data that
+        should be analyzed that should have the same shape as the images.
+
+    distance : float, optional
+        Cutoff distance for finding adjacent voxels.
```

Comment:
```suggestion
        Cutoff distance for finding adjacent voxels. Default 1.99 
```

Where does the number 1.99 come from?

Reply:
It is there so that the default produces a 3x3 patch (I suppose < 2 would be fine, but I just used 1.99 to be 'safe')

Reply:
If this is voxels, I wonder if you just want to fix it to 1.99 or 2, so that adjacency means one and only one thing.

Reply:
Voxels, not mm, and I believe the user should be able to select the number of adjacent voxels in a neighbourhood of specified size. This is certainly what I need to do at the moment

Reply:
Additionally, I was told previously to leave default values out of the documentation, since it inevitably gets missed later and leads to docs errors?